Website/Packages/datax-pipeline/package.json

```diff
@@ -21,7 +21,9 @@
     "dev": "rimraf dist && webpack --mode development --config webpack.config.js --bail --progress --display-modules --display-error-details --colors --display-chunks --display-reasons",
     "devpatch": "rimraf dist && webpack --mode development --config webpack.config.js --bail --progress --display-modules --display-error-details --colors --display-chunks --display-reasons && localdevpatch.bat"
   },
-  "dependencies": {},
+  "dependencies": {
+    "react-datetime-picker": "^2.4.0"
```

Comment:
Remove the ^ from the version. We want to lock down to this specific version that you have tested as opposed to opening up the versions to picking up latest version which may have bugs or incompabilities.

Reply:
updated.